### PR TITLE
Round up reading time

### PR DESCRIPTION
**Pull request by elbaro**
**IMPORTANT: Please do not create a Pull Request adding a new feature without discussing it first.**

The place to discuss new features is the forum: <https://zola.discourse.group/>
If you want to add a new feature, please open a thread there first in the feature requests section.

Sanity check:

* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?

## Code changes
(Delete or ignore this section for documentation changes)

* [x] Are you doing the PR on the `next` branch?

If the change is a new feature or adding to/changing an existing one:

* [x] Have you created/updated the relevant documentation page(s)?

Medium has no '0 min read', so it should be `max(1, read_time)` or `ceil(read_time)`. This PR rounds up.

### CODE REVIEW

File: components/utils/src/site.rs [hunk 1]
```diff
@@ -86,11 +86,18 @@ mod tests {
         assert!(res.is_err());
     }
 
+    #[test]
+    fn reading_analytics_empty_text() {
+        let (word_count, reading_time) = get_reading_analytics("  ");
+        assert_eq!(word_count, 0);
+        assert_eq!(reading_time, 0);
```
Comment (Keats) [inline components/utils/src/site.rs:93]: shouldn't that be one according to your PR comment?

Comment (elbaro) [reply components/utils/src/site.rs:93]: Didn't realize Medium allows a title-only post. Just tested in Medium and it is 1 min read, although I think 0 min read makes more sense. What do you prefer?

Comment (Keats) [reply components/utils/src/site.rs:93]: 0 makes more sense yes.

**Comment by Keats** [comment]
Thanks!

**Pull request merged**